.github/pull_request_template.md

```diff
@@ -1,13 +1,5 @@
-[PMM-XXXX](https://jira.percona.com/browse/PMM-XXXX)
+https://jira.percona.com/browse/PMM-XXXX
```

Comment:
Let's use autolinks instead: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources